subsys/bluetooth/controller/ll_sw/ll_adv.c

```diff
@@ -443,6 +443,7 @@ u8_t ll_adv_enable(u8_t enable)
 #endif /* CONFIG_BT_CTLR_ADV_EXT */
 	} else {
 		bool priv = false;
+
```

Comment:
Is this critical change? (just this alone in this file?)

Reply:
Reverted.